frontend/src/components/editor/actions/useCellActionButton.tsx

```diff
@@ -178,6 +183,18 @@ export function useCellActionButtons({ cell }: Props) {
           formatEditorViews({ [cellId]: editorView }, updateCellCode);
         },
       },
+      {
+        icon: <MarkdownIcon />,
+        label: "Markdown mode",
```

Comment:
```suggestion
        label: "View as Markdown",
```

Reply:
Done I think -- used the same format for the `View as Python` when toggling back, lmk if you have a diff preference